stacks/container-apps/root.yml

```diff
@@ -281,3 +288,70 @@ Resources:
           Value: !Ref AWS::StackId
       TemplateURL: !Join ["", [!Ref TemplateUrlPrefix, "web-application.yml"]]
       TimeoutInMinutes: 5
+  RemixStack:
+    Condition: IsStaging
+    Type: "AWS::CloudFormation::Stack"
+    Properties:
+      NotificationARNs:
+        - Fn::ImportValue:
+            !Sub "${InfrastructureNotificationsStackName}-CloudFormationNotificationSnsTopic"
+      Parameters:
+        VPC: !Ref VPC
+        PlatformALBDNSName: !Ref PlatformALBDNSName
+        PlatformALBFullName: !Ref PlatformALBFullName
+        PlatformALBCanonicalHostedZoneID: !Ref PlatformALBCanonicalHostedZoneID
+        PlatformALBHTTPListenerArn: !Ref PlatformALBHTTPListenerArn
+        PlatformALBHTTPSListenerArn: !Ref PlatformALBHTTPSListenerArn
+        ECSCluster: !Ref ECSCluster
+        ECSServiceAutoscaleIAMRoleArn: !Ref ECSServiceAutoscaleIAMRoleArn
+        ECSServiceIAMRole: !Ref ECSServiceIAMRole
+        OpsDebugMessagesSnsTopicArn:
+          Fn::ImportValue:
+            !Sub "${InfrastructureNotificationsStackName}-OpsDebugMessagesSnsTopicArn"
+        OpsErrorMessagesSnsTopicArn:
+          Fn::ImportValue:
+            !Sub "${InfrastructureNotificationsStackName}-OpsErrorMessagesSnsTopicArn"
+        EnvironmentType: !Ref EnvironmentType
+        EnvironmentTypeAbbreviation: !Ref EnvironmentTypeAbbreviation
+        EcrRegion: !Ref EcrRegion
+        SecretsBase: !Ref SecretsBase
+        ContainerMemory: !Ref ContainerMemory
+        ContainerMemoryReservation: !Ref ContainerMemoryReservation
+        ContainerCpu: !Ref ContainerCpu
+        PlatformALBListenerPriorityPrefix: !Ref RemixPlatformALBListenerPriorityPrefix
+        EcrImageTag: !Ref RemixEcrImageTag
+        SecretsVersion: !Ref RemixSecretsVersion
+        AppName: "remix"
+        CreateWorker: "true"
+        DesiredWorkersProduction: "0"
+        DesiredWorkers: "0"
+        ContainerPort: "3000"
+        HealthCheckPath: "/api"
+      Tags:
+        - Key: "prx:cloudformation:stack-name"
+          Value: !Ref AWS::StackName
+        - Key: "prx:cloudformation:stack-id"
+          Value: !Ref AWS::StackId
+      TemplateURL: !Join ["", [!Ref TemplateUrlPrefix, "web-application.yml"]]
+      TimeoutInMinutes: 5
+  RemixScheduledTaskStack:
+    Condition: IsStaging
+    Type: "AWS::CloudFormation::Stack"
+    Properties:
+      NotificationARNs:
+        - Fn::ImportValue:
+            !Sub "${InfrastructureNotificationsStackName}-CloudFormationNotificationSnsTopic"
+      Parameters:
+        ECSCluster: !Ref ECSCluster
+        ECSServiceIAMRole: !Ref ECSServiceIAMRole
+        ScheduleCommand: "[\"load_episodes\"]"
+        ScheduleExpression: "rate(1 hour)"
+        ScheduleTaskDefinitionName: "remix-worker"
```

Comment:
Where does this come from? It seems like it needs to exist for the task def. before the `containerOverrides`, but it only looks like it exists in the context of the events rule

Reply:
Added a `DependsOn` to make this more explicit

Reply:
Oh I see, the `AppName` is used to derive that in `web-application.yml`. I think what I would do is, export either the AppName from the app stack, or export the entire worker name from the app stack (just as a sub that matches the one that makes the name in the task def), and then `GetAtt` that like you do for the Arn. I'd also get rid of the `DependsOn`; those shouldn't get used unless we explicitly need to control the order Cfn does things.